data-share/data-share-service/src/main/java/io/mosip/datashare/util/PolicyUtil.java

```diff
@@ -108,4 +119,42 @@ public void emptyPartnerPolicyCache() {
 
 	}
 
+	/**
+	 * Provides static data share policy for sharing the data.
+	 * @param policyId Policy Id from request
+	 * @param subscriberId Subscriber Id from request
+	 * @return the DataShareDto object
+	 */
+	public DataShareDto getStaticDataSharePolicy(String policyId, String subscriberId) {
+		LOGGER.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.POLICYID.toString(),
+				policyId, "PolicyUtil::getStaticDataSharePolicy()::entry");
+		try {
+			String staticPolicyId = env.getProperty(STATIC_DATA_SHARE_POLICY_ID);
+			if (staticPolicyId == null)
+				throw new StaticDataShareException("Please configure the static data share policy Id");
+
+			String staticSubscriberId = env.getProperty(STATIC_DATA_SHARE_SUBSCRIBER_ID);
+			if (staticSubscriberId == null)
+				throw new StaticDataShareException("Please configure the static data share subscriber Id");
+
+			if (!staticPolicyId.equals(policyId) || !staticSubscriberId.equals(subscriberId))
+				throw new StaticDataShareException("Either Policy Id or Subscriber Id not matching with configured in system");
+
+			String staticSharePolicy = env.getProperty(STATIC_DATA_SHARE_POLICY);
+			if (staticSharePolicy == null)
+				throw new StaticDataShareException("Please configure the static data share policy");

```

Comment:
Do we really need a new exception class ? Can we not use the existing PolicyException ?

Reply:
Done. Removed the StaticDataShareException.java and now using the PolicyException.java.